step.yml

```diff
@@ -0,0 +1,54 @@
+
+title: |
+  Create ZIP
+summary: |
+  Create ZIP
+description: |
+  Create ZIP
+website: https://github.com/bitrise-steplib/steps-create-zip
+source_code_url: https://github.com/bitrise-steplib/steps-create-zip
+support_url: https://github.com/bitrise-steplib/steps-create-zip
+host_os_tags:
+  - osx-10.10
+  - ubuntu-16.04
+
+type_tags:
+  - utility
+
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+
+
+toolkit:
+  go:
+    package_name: github.com/birmachera/bitrise-step-create-zip
+
+
+inputs:
+  - source_path: ./
```

Comment:
Please remove these default input values. If a user mistakenly doesn't change this value then we start compress his whole source dir. Also it must be specified exactly,  so better to leave it empty and as a required field